metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran_log_api.py

```diff
@@ -76,7 +77,7 @@ def _initialize_fivetran_variables(
         )
 
     def _query(self, query: str) -> List[Dict]:
-        logger.debug(f"Query : {query}")
+        logger.debug("Query : {}".format(query))
```

Comment:
why was this changed?

Reply:
Changes happen automatically. Let me revert it